src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindOneAndReplaceIntegrationTest.java

```diff
@@ -176,7 +175,7 @@ public void byIdWithIdNoChange() {
           .post(CollectionResource.BASE_PATH, namespaceName, collectionName)
           .then()
           .statusCode(200)
-          .body("data.docs", hasSize(0))
```

Comment:
Could maybe change to check `hasSize(1)` (to prevent accepting returning multiple documents)